en/08_Changelogs/6.0.0.md

```diff
@@ -41,6 +44,58 @@ title: 6.0.0 (unreleased)
   - [`getSchemaDataDefaults()` now includes attributes](#formfield-schema-data)
 - [Full list of removed and changed API (by module, alphabetically)](#api-removed-and-changed)
 
+## Change to commercially supported modules {#changes-to-support}
+
+Some Silverstripe CMS modules are commercially supported. Silverstripe commits to looking after those modules for the duration of the Silverstripe CMS 6 lifecycle.
```

Comment:
```suggestion
Some Silverstripe CMS modules are commercially supported. Silverstripe Ltd. commits to looking after those modules for the duration of the Silverstripe CMS 6 lifecycle.
```

Reply:
Silverstripe (without ltd) is correct. See https://docs.silverstripe.org/en/5/contributing/documentation/#references-to-various-things